### PR TITLE
create front end pages [#160163573]

### PR DESCRIPTION
**Pull request by mozzy22**
create front end page for the app

### CODE REVIEW

**Comment by mulondo** [comment]
I see you are utilizing git and pivotal tracker very well, keep it keep.

File: html/Front.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      </div>
+
+
+      <div  id = "form">
+      <form method = "POST">
+          <fieldset id = "field">
+              <legend>Login</legend>
+          <div>
+              <div class="input">
+                 <spnan> First name:</span>
```
Comment (jeanjoe) [inline html/Front.html]: check the spelling of your opening span tag. should have been `<span>`

File: html/Front.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    <input class="inputbutton1" type = "text" name = "first" placeholder= "first name"></input>
```
Comment (jeanjoe) [inline html/Front.html]: input tags are self closing tags meaning u don't need `</input>` to close it either leave it as it is or end it with` />`

File: html/Front.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>index</title>
+    <link rel="stylesheet" href="../css/front.css">
+    
+    <meta charset="UTF-8"> 
+    <meta name="description" content="Front End for a level-up Events app">
+    <meta name="keywords"  lang"en-us"  content="events, event, app, level-up, andela">
```
Comment (jeanjoe) [inline html/Front.html]: Your are missing `=` inside your meta tag for `lang"en-us"`

**Comment by mozzy22** [comment]
can u merge all the commits into one using squash commits?

**Comment by jeanjoe** [comment]
I don't think you can merge commits but u can add a comment when you are making a PR to merge the commits to another branch.

The previous commits cannot be overridden.

**Comment by alexkayabula** [comment]
Good stuff

**Pull request closed without merging**